cfme/utils/template/base.py

```diff
@@ -131,10 +137,12 @@ def raw_image_url(self):
 
     @property
     def image_name(self):
-        """ Returns filename of an image.
+        """ Returns filename of an downloaded or unzipped image.
 
         Example output: cfme-type-version-x86_64-vhd
         """
+        if self._unzipped_file:
```

Comment:
can just be `return self._unzipped_file or self.raw_image_url.split('/')[-1]`

Reply:
Done